test/acceptance/behave/components/ngsiv2/api_entry_point/retrieve_api_resource.feature

```diff
@@ -45,7 +45,7 @@ Feature: general operations in Context Broker using NGSI API v1 and v2
   Actions After the Feature:
   Setup: stop ContextBroker
 
-  @version
+  @version @skip
```

Comment:
I will remove the skip tag, and I will skip this scenario in execution time with `-t=-version`. This way not generate confusion .